[DOCS] ExpectAI approval workflow (#11072)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/expectations/manage_expectations.md

1. In GX Cloud, click **Data Assets**.
2. In the **Data Assets** list, click the Data Asset name.
3. Click **Generate Expectations**.
   :::note This might take a few minutes
   ExpectAI may take a few minutes to analyze your data and recommend personalized Expectations. You can navigate away from the page while ExpectAI works in the background. GX will email you when your recommended Expectations are ready for review.
   :::

> 4. Review the generated Expectations and **Approve** (✓) or **Reject** (✗) them within 48 hours. After that, the recommendations will be discarded.

Reviewer: ```suggestion
4. Review the generated Expectations and **Approve** (✓) or **Reject** (✗) them within 48 hours. After that, any undecided recommendations will be discarded.
```
My suggestion here isn't the best wording, but I'm wondering if we need to highlight that it's only the recommendations in the "non-approved & non-rejected" state that will be discarded?

That could be covered by the antecedent use of "recommendations" rather than "Expectations," but I wonder if people will possibly read it as discarding ExpectAI things they've already approved/rejected.

Author: Hmmm I see what you mean but something about "undecided" feels off to me - like it's describing the recommendation's state of mind rather than a lack of user action regarding the recommendation. Probably I just have a strong association of the word "undecided" and the term "undecided voter" in my mind. I'll make a different change in the spirit of this suggestion. 

```suggestion
4. Review the recommended Expectations and **Approve** (✓) or **Reject** (✗) them within 48 hours. After 48 hours, any remaining recommendations will be discarded.
```

Reviewer: @klavavej Sorry I missed this! I didn't get the notification from github. And yes, totally agree with the associations between "undecided" and "undecided voter". I like the proposed changes you've got here.

Author: thanks! if everything looks good now, would you please approve this PR?
